simplq/src/components/pages/Admin/AdminPage.jsx

```diff
@@ -7,7 +7,7 @@ import { selectQueueName } from 'store/selectedQueue';
 import Ribbon from 'components/common/Ribbon';
 import Tour from 'components/common/Tour';
 import { useGetSelectedQueue } from 'store/asyncActions';
-import HeaderSection from './AdminHeaderSection';
+import HeaderSection from './HeaderSection';
```

Comment:
> By convention, a file that exports only one class, function, or constant should be named for that class, function or constant. Anything else may confuse maintainers.

In this case, the class name should be changed, not the file name.

Reply:
@mradenovic I thought it was funky that it was having me change the file name. I appreciate the explanation.